src/sentry/static/sentry/app/components/createAlertButton.tsx

```diff
@@ -391,7 +393,7 @@ const CreateAlertButton = withApi(
         </Button>
       );
 
-      const showGuide = !organization.alertsMemberWrite;
+      const showGuide = !organization.alertsMemberWrite && !!showPermissionGuide;
```

Comment:
not sure what is preferred here, if I don't do the `!!` in front of `showPermissionGuide` then `showGuide`'s type is `boolean | undefined` which makes me a little uncomfy, but technically the behavior wouldn't be changed so perhaps the double bang is unnecessary.